src/dashboard/frontend/src/components/Stage/cockpit/cockpitBody.module.css

```
@@ -77,13 +78,110 @@
+    border-color: transparent;
+    background: transparent;
+    padding: 4px;
+  }
+
+  .spineCollapsed [data-section="verification-gates"] > :first-child,
+  .spineCollapsed [data-section="verification-gate"] > div {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)